core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -53,28 +37,9 @@ import scala.jdk.CollectionConverters._
  * easier to migrate away from `ZkUtils` (since removed). We should revisit this. We should also consider whether a
  * monolithic [[kafka.zk.ZkData]] is the way to go.
  */
-class KafkaZkClient private[zk] (
-  zooKeeperClient: ZooKeeperClient,
-  isSecure: Boolean,
-  time: Time,
-  enableEntityConfigControllerCheck: Boolean
-) extends AutoCloseable with Logging {
-
-  private val metricsGroup: KafkaMetricsGroup = new KafkaMetricsGroup("kafka.server", "ZooKeeperClientMetrics")
```

Comment:
@m1a2st please include those removed zk-related metrics in #18365

Reply:
Sure, I will addressed it.